node/service/src/lib.rs

```diff
@@ -610,12 +610,7 @@ macro_rules! chain_ops {
 pub fn new_chain_ops(
 	config: &Configuration,
 ) -> Result<
-	(
-		Arc<Client>,
-		Arc<FullBackend>,
-		sc_consensus::BasicQueue<Block, PrefixedMemoryDB<BlakeTwo256>>,
```

Comment:
This change comes from substrate [PR #14612](https://github.com/paritytech/substrate/pull/14612)